install_instructions/client/linux-arm7.md

```diff
@@ -1,11 +1,169 @@
 ---
-lastUpdated: "2021-03-22 11:20:18"
+lastUpdated: "2021-05-03 12:06:00"
 streamPiVersion: "1.0.0"
-editedBy: "j4ckofalltrades"
+editedBy: "rnayabed"
 ---
 
-# Setup Stream-Pi on Linux
+# Setup Stream-Pi on Linux ARMv7 (32 Bit) (Raspberry Pi)
+
+The instructions below will go over how to set up **Stream-Pi Client** on a **Linux ARMv7** device.
+Examples of Linux ARMv7 devices include **Raspberry Pi** running on **Raspberry Pi OS 32-bit**. 
+**This is NOT to be followed if your Raspberry Pi is running a 64 bit OS.**
```

Comment:
If the instructions are different for Raspberry Pi 64 bit as opposed to a 32 bit then we need to discuss how to break that up, because things should be as easy for the end user as possible

Reply:
There are actually no builds available for Pis running 64 Bit OS.From next build however, i will be rolling out ARMv6 builds which should be backwards compatible with both ARMv7, and aarch64

When that happens, the instructions for 32-bit and 64-bit ARM linux will be the SAME. Until then, we HAVE to let them know that this build does not work on a 64 bit ARM OS, just 32 bit (Armv7)